app/models/partner.rb

```diff
@@ -82,37 +82,15 @@
 #  diaper_funding_source      :string
 #  created_at                 :datetime         not null
 #  updated_at                 :datetime         not null
-#  email                      :string           default(""), not null
-#  encrypted_password         :string           default(""), not null
-#  reset_password_token       :string
-#  reset_password_sent_at     :datetime
-#  remember_created_at        :datetime
-#  sign_in_count              :integer          default(0), not null
-#  current_sign_in_at         :datetime
-#  last_sign_in_at            :datetime
-#  current_sign_in_ip         :string
-#  last_sign_in_ip            :string
-#  invitation_token           :string
-#  invitation_created_at      :datetime
-#  invitation_sent_at         :datetime
-#  invitation_accepted_at     :datetime
-#  invitation_limit           :integer
-#  invited_by_type            :string
-#  invited_by_id              :bigint(8)
-#  invitations_count          :integer          default(0)
 #  other_agency_type          :string
 #
 
 class Partner < ApplicationRecord
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
   ACTIVE_FAMILY_REQUESTS = [1, 3, 27].freeze
 
-  devise :invitable, :database_authenticatable,
-         :recoverable, :rememberable, :trackable, :validatable
-
   include DiaperBankClient
 
+  has_one :user
```

Comment:
For now I decided to leave this as a `has_one`. My goal with this PR was to replicate the exact experience that we currently have, while setting ourselves up with a database design that will allow us to improve on this further in future work. (`has_many :users`)

Reply:
May want to ignore my other comment about adding multiple users via the diaper app.

Reply:
Do we want a dependent destroy on the user(s)?